modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/order/DefaultCommerceOrderValidatorImpl.java

```diff
@@ -78,12 +78,8 @@ public CommerceOrderValidatorResult validate(
 			cpInstance);
 		int maxOrderQuantity = cpDefinitionInventoryEngine.getMaxOrderQuantity(
 			cpInstance);
-		String[] allowedOrderQuantities =
```

Comment:
why is the back order check removed?

Reply:
Source formatter complained about it

Reply:
Then why you didn't remove this check also in the other validate method of this class?

Reply:
Yes, fixed. Thanks!